packages/block-library/src/classic/edit.js

```diff
@@ -112,6 +121,20 @@ export default class ClassicEdit extends Component {
 			bookmark = null;
 		} );
 
+		editor.on(
+			'Paste Change input Undo Redo',
+			debounce( () => {
```

Comment:
Why are you using `debounce` here?

Reply:
We figured that this could be a busy call as someone is typing, making lots of changes, etc., and we don't need ervy minute change to go through this cycle. We didn't actually start to see any visible performance degradation until you got upwards of 50,000 words in a post, but didn't think it would hurt to add this in anyway.

Reply:
> we didn't think it would hurt to add this in anyway

I believe that too and I also think that `250` is a fair debounce time.

It's great that you have made some tests checking performance! :+1: